src/wms/src/test/java/org/geoserver/wms/animate/AnimatorTest.java

```diff
@@ -132,7 +132,11 @@ public void testAnimator() throws Exception {
         checkAnimatedGif(requestURL, false);
         checkAnimatedGif(requestURL + "&format_options=gif_loop_continuously:true", true);
         checkAnimatedGif(requestURL + "&format_options=gif_loop_continuosly:true", true);
-
+        // check all valid disposal methods
+        checkAnimatedGif(requestURL + "&format_options=gif_disposal:none", false);
```

Comment:
The test is not really checking the disposal method got applied, could you add that?